mountpoint-s3-client/examples/client_benchmark.rs

```diff
@@ -238,9 +239,11 @@ fn create_s3_client_config(region: &str, args: &CliArgs, nics: Vec<String>) -> S
     config = config.throughput_target_gbps(args.throughput_target_gbps);
     config = config.memory_limit_in_bytes(args.crt_memory_limit_gb * 1024 * 1024 * 1024);
     config = config.network_interface_names(nics);
-
     config = config.part_size(args.part_size);
```

Comment:
can these be chained into one statements? not necessarily also chained with declaration but worth considering too. Also applies to the following `args.enable_backpressure` block